src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Text_Format_PowerFxV1CompatDisabled.txt

```diff
@@ -8,18 +8,586 @@ Error({Kind:ErrorKind.InvalidArgument})
 
 // If have empty language code, throw error
 >> Text(1234.5678,"[$-]")
-Error({Kind:ErrorKind.BadLanguageCode})
+Error({Kind:ErrorKind.InvalidArgument})
 
 // If not closed by a ']', throw error
 >> Text(1234.5678,"[$-a")
-Error({Kind:ErrorKind.BadLanguageCode})
+Error({Kind:ErrorKind.InvalidArgument})
 
 // If not closed by a ']', throw error
 >> Text(1234.5678, "[$-en-US#,##0.00", "en-US")
-Error({Kind:ErrorKind.BadLanguageCode})
+Error({Kind:ErrorKind.InvalidArgument})
 
 >> Text(1234.5678, "[$-]fr-FR # ##0,00", "vi-VI")
-Error({Kind:ErrorKind.BadLanguageCode})
+Error({Kind:ErrorKind.InvalidArgument})
 
 >> Text(1234.5678, "[$-fr-FR][$-en-US] # ##0,00", "vi-VI")
+Error({Kind:ErrorKind.InvalidArgument})
+
+// end with e or e+ in numeric is invalid
+>> Text(1234567.1234567, "0.00e+")
+Error({Kind:ErrorKind.InvalidArgument})
+
+>> Text(1234567.1234567, "0.00e")
+Error({Kind:ErrorKind.InvalidArgument})
+
+>> Text(0, "longdate")
+Error({Kind:ErrorKind.InvalidArgument})
+
+>> Text(0, "shortdate")
+Error({Kind:ErrorKind.InvalidArgument})
+
+>> Text(DateTime(1970, 1, 1, 14, 32, 25, 678), "longdatetime")
```

Comment:
Should we change it the behavior with Power Fx V1 disabled?

Reply:
the blocking unsupported character like this is happened at runtime and at runtime i don't think we check PowerFxV1 flag. Do we want to check flag for all blocking and only do it when flag enable?